lib/code_corps/helpers/policy.ex

```diff
@@ -0,0 +1,67 @@
+defmodule CodeCorps.Helpers.Policy do
```

Comment:
Added this one and it works nicely. The only policy that doesn't rely on it completely is the one for organization memberships, but that was to be expected to a degree, since it ought to be the most complex. Still, even that one relies on this module a great deal.